examples/create-react-app/README.md

```diff
@@ -2,7 +2,7 @@
 
 This shows the usage of CMS design system components in a [create-react-app](https://create-react-app.dev/docs/getting-started/) project.
 
-Refer to [install with NPM](https://design.cms.gov/startup/installation/) for instructions on how to install the design system as a dependency package. The `.env` is for a basic create-react-app confirguration with the design system. The [`/src/styles/index.scss`](https://design.cms.gov/startup//sass-and-css/) includes the design system's styles and paths for the image and font directories.
+Refer to [install with NPM](https://design.cms.gov/startup/installation/) for instructions on how to install the design system as a dependency package. The `.env` is for a basic `create-react-app` confirguration with the design system. The [`/src/styles/index.scss`](https://design.cms.gov/startup/sass-and-css/) includes the design system's styles and paths for the image and font directories.
```

Comment:
Configuration is spelled wrong, the link to the sass and css page is incorrect. i would search for {{root}} to see an example of how the url should look. also, since you are specifically linking to the section for fonts and images, please use # to directly link to that section of the page

Reply:
Thanks, fixed typo and `#` link to fonts/images. As for {{root}}, i didn't use it because the `.md` is viewed on github and not on a web page, hence i found {{root}} is not working, so had to refer to the full url path.

Reply:
Ah right, {root}} is right